src/ui/ResponsiveComponents/ResponsiveTabs.ts

```diff
@@ -329,13 +329,9 @@ export class ResponsiveTabs implements IResponsiveComponent {
   }
 
   private positionPopup() {
-    PopupUtils.positionPopup(
-      this.dropdownContent.el,
-      this.dropdownHeader.el,
-      this.coveoRoot.el,
-      { horizontal: PopupHorizontalAlignment.INNERRIGHT, vertical: PopupVerticalAlignment.BOTTOM },
-      this.coveoRoot.el
-    );
+    this.coveoRoot.el.appendChild(this.dropdownContent.el);
+    const options: PopperJs.PopperOptions = { placement: 'bottom-end' };
```

Comment:
If `options` end up still being this short, it would be fine to put it in the object directly.

Reply:
Fair point. I ended up reverting from popper back to our current positioning library, since the positioning does not work for screen-sizes between `480px` and `830px`. I tested with just the CSS modifications and it works.